bi_sql_editor/models/bi_sql_view.py

```diff
@@ -570,3 +570,9 @@ def _refresh_size(self):
                 sql_view.view_name)
             self.env.cr.execute(req)
             sql_view.size = self.env.cr.fetchone()[0]
+
+    @api.multi
+    def button_preview_sql_expression(self):
+        self.button_validate_sql_expression()
+        res = self._execute_sql_request()
+        raise UserError('\n'.join(map(lambda x: str(x), res[:100])))
```

Comment:
Hi.
This button is raising an error, even if all is ok ? 

not sure to understand. Could you precise ? 

thanks !

Reply:
If it was done on purpose to display in the window popup the result of the SQL query.